Examples/Cxx/Examples.h

```diff
@@ -308,10 +308,10 @@ namespace CIGREMV {
         Real phiPLLInit = 8e-06;
         Real pInit = 450000.716605;
         Real qInit = -0.577218;
-        Real phi_dInit = 3854.197405*scaling_I;
-        Real phi_qInit = -0.003737*scaling_I;
-        Real gamma_dInit = 128.892668*scaling_I;
-        Real gamma_qInit = 23.068682*scaling_I;
+        Real phi_dInit = 3854197405*scalingKi;
```

Comment:
The new values have been determined by simulation, correct?

Reply:
These new values have just been recalculated for the new definition of scalingKp and scalingKi. In fact, they have not changed.